README.md

```diff
@@ -38,6 +38,7 @@ Project Layout
 * [`design`](design): Design documentation. See the [design/README.md](design/README.md) for details about building / viewing.
 * `codegen-server`: Whitelabel Smithy server code generation
 * `codegen-server-test`: Smithy protocol test generation & integration tests for Smithy server whitelabel code
+* `examples`: A collection of server implementation examples
```

Comment:
I think this folder should be called `server-examples` or `examples-server` to be inline with the rest of the folder structures.

Reply:
Sounds good to me.